web/views.py

```diff
@@ -13,7 +14,8 @@ def about(request):
 	return render(request, 'pages/about.html',{})
 
 def members(request):
-	return render(request, 'pages/members.html',{})
+	members_list = UserProfile.objects.filter(active=True).order_by('activity_status')
```

Comment:
Suggestion: instead of activity_status ordering (which you're not using), you could sort alphabetically by first name :)